tests/python/unittest/test_optimizer.py

```diff
@@ -396,6 +397,7 @@ def update(self, index, weight, grad, state):
                 weight32[:] += -lr * (grad32 + wd * weight32)
             else:
                 weight32[:] += (self.momentum**2 * mom) - lr*(self.momentum+1)*(grad32 + wd*weight32)
+                mom = (self.momentum*mom) - lr*(grad32 + wd*weight32)
```

Comment:
`lr` should be outside of the momentum state for mxnet standard practice.

Reply:
why? but that is not nesterov logic right?

![image](https://user-images.githubusercontent.com/2778341/64295137-695db800-cf25-11e9-880e-250298b2df12.png)

Reply:
Please see SGD implementation:
http://mxnet.incubator.apache.org/api/python/optimization/optimization.html?highlight=optimizer.sgd#mxnet.optimizer.SGD

Reply:
the sgd unit test also has `lr` as part of `mom` update - https://github.com/apache/incubator-mxnet/blob/master/tests/python/unittest/test_optimizer.py#L142. 

let me know if you still think this needs to change.

Reply:
Thanks! You are right. The MXNet implementation is puting lr inside. The update should be:


![image](https://user-images.githubusercontent.com/8041160/64364156-2c4a0200-cfc7-11e9-92d6-331d8b656d73.png)
![image](https://user-images.githubusercontent.com/8041160/64364165-2f44f280-cfc7-11e9-9416-b092fd294294.png)

I will check your implementation again. Thanks!